common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -161,3 +161,35 @@ def test_demand_hint(self):
                 {'event': {u'hint_index': 0, u'hint_len': 2, u'hint_text': u'demand-hint1'}}
             ],
             actual_events)
+
+
+class ProblemPartialCredit(ProblemsTest):
+    """
+    Makes sure that the partial credit is appearing properly.
+    """
+    def get_problem(self):
```

Comment:
I don't see this method being called by the test_ method. Should this method be part of setUp? Or removed?